sql/item_subselect.cc

```diff
@@ -3298,6 +3299,14 @@ bool Item_exists_subselect::exists2in_processor(void *opt_arg)
     set possible optimization strategies
   */
   in_subs->emb_on_expr_nest= emb_on_expr_nest;
+
+  {
```

Comment:
I don' think there an need for anonymous block here. If there is, indent its content.

Reply:
There is, purely due to how OPT_TRACE_TRANSFORM.  It's not really at another lexical level though.